### PR TITLE
Master

### PR DESCRIPTION
**Pull request by kit-91-daniil**
Hey, TA-user1, please, check out these changes)

### CODE REVIEW

File: master_branch_file.py [hunk 1]
```diff
@@ -0,0 +1,4 @@
+"""
+
```
Comment (TA-user1) [inline master_branch_file.py:2]: Why here is blank line. Delete it please.

File: unuseful_file.txt [hunk 2]
```diff
@@ -1 +1,4 @@
 Here is some changes in unuseful file)
```
Comment (TA-user1) [inline unuseful_file.txt:1]: Ok, let's leave it here

File: file1.txt [hunk 3]
```diff
@@ -1,2 +1,3 @@
 file1
 Welcome friend!
+This string was written directly on github editor.
```
Comment (TA-user1) [inline file1.txt:3]: agree

**Review by TA-user1 — APPROVED** [review]

File: unuseful_file.txt (hunk 2 above)
Comment (kit-91-daniil) [reply unuseful_file.txt:1]: Nice)

File: file1.txt (hunk 3 above)
Comment (kit-91-daniil) [reply file1.txt:3]: Nice

File: master_branch_file.py (hunk 1 above)
Comment (kit-91-daniil) [reply master_branch_file.py:2]: You are basta..d, get out from our conf

File: file1.txt (hunk 3 above)
Comment (kit-91-daniil) [reply file1.txt:3]: !

File: master_branch_file.py (hunk 1 above)
Comment (kit-91-daniil) [reply master_branch_file.py:2]: !

File: master_branch_file.py [hunk 4]
```diff
@@ -0,0 +1,4 @@
...
+This file was made after creating develop and hotfix branches
+"""
```
[the remark this answers is not in the document]
Comment (kit-91-daniil) [reply master_branch_file.py:4]: !

File: unuseful_file.txt (hunk 2 above)
Comment (kit-91-daniil) [reply unuseful_file.txt:1]: !

**Pull request merged**